src/main/java/org/cryptomator/ui/unlock/UnlockWorkflow.java

```diff
@@ -85,6 +100,14 @@ private void handleIllegalMountPointError(IllegalMountPointException impe) {
 		});
 	}
 
+	private void handleFuseRestartRequiredError(FuseRestartRequiredException fRRE) {
+		Platform.runLater(() -> {
+			window.setScene(fuseRestartRequiredScene.get());
+			window.setTitle(String.format(resourceBundle.getString("unlock.error.fuseRestartRequired.title"), vault.getDisplayName()));
```

Comment:
Instead of injecting and using the resource bundle in the UnlockWorkflow, can you inject and use it in the fuseRestartRquiredController?

JavaFX offers the initialize method to for setup:
```java
public void initialize() {
    window.setTitle(resourceBundle.getString(....));
}
```